Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
             targets: ["Benchmark"])
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.0.1"),
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.4.4"),
```

Comment:
Can you split this out into a separate change please?

Reply:
It looked like old version of SAP could fail to build with Swift 5.4 on Windows for some reason. But I've reverted the update in `Package.swift` and `Package.resolved` for now as you've requested. Curious to see if CI passes on Windows without updating the SAP dependency. Could you approve it for another CI run? I'm not able to trigger it with a sole push.